test/e2e_mc/multicluster_test.go

```diff
@@ -0,0 +1,191 @@
+// Copyright 2023 Antrea Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e_mc
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"antrea.io/theia/test/e2e"
+)
+
+const (
+	iperfTimeSec = 12
+	// Set target bandwidth(bits/sec) of iPerf traffic to a relatively small value
+	// (default unlimited for TCP), to reduce the variances caused by network performance
+	// during 12s, and make the throughput test more stable.
+	iperfBandwidth = "10m"
+)
+
+func TestMultiCluster(t *testing.T) {
```

Comment:
Should we also check the number of records? like https://github.com/antrea-io/theia/blob/bc7a0a0a175434d6ea757d66539afd055b35cb5c/test/e2e/flowvisibility_test.go#L786

Reply:
Thanks for reviewing. I've updated it to check the total number of flow records instead of just checking whether the number of flow records is greater than 0.